modules/tensor_mechanics/doc/content/modules/tensor_mechanics/index.md

```diff
@@ -111,10 +107,9 @@ underlying systems:
 - The current system based on the [StressDivergenceTensors](/StressDivergenceTensors.md) and related kernels.
 - A newer system based on the [TotalLagrangianStressDivergence](/TotalLagrangianStressDivergence.md) and [UpdatedLagrangianStressDivergence](/UpdatedLagrangianStressDivergence.md) kernels.
 
-The current system is fully compatible with the entirety of the Tensor Mechanics and MOOSE ecosystems.  However, it suffers from convergence issues caused by
-non-exact Jacobians when not used with the Automatic Differentiation variants of the kernels and underlying materials.
-
-The newer system (referred to in the documentation as the *Lagrangian* kernels) has exact Jacobians, but is a work in progress.  Currently, it supports
+The current system suffers from convergence issues caused by
```

Comment:
That's a bit strong. Maybe qualify that with wither "may suffer" or "for large deformations"?

Reply:
Yep, makes sense.